presto-raptor-legacy/src/test/java/io/prestosql/plugin/raptor/legacy/TestRaptorIntegrationSmokeTestMySql.java

```diff
@@ -26,25 +26,35 @@
 
 import static io.prestosql.plugin.raptor.legacy.RaptorQueryRunner.copyTables;
 import static io.prestosql.plugin.raptor.legacy.RaptorQueryRunner.createSession;
+import static java.lang.String.format;
 
 @Test
 public class TestRaptorIntegrationSmokeTestMySql
         extends TestRaptorIntegrationSmokeTest
 {
-    private TestingMySqlServer mysqlServer;
+    private MySQLContainer<?> mysqlContainer;
 
     @Override
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        mysqlServer = new TestingMySqlServer("testuser", "testpass", "testdb");
-        return createRaptorMySqlQueryRunner(mysqlServer.getJdbcUrl("testdb"));
+        mysqlContainer = new MySQLContainer<>("mysql:8.0.12");
+        mysqlContainer.start();
+        return createRaptorMySqlQueryRunner(getJdbcUrl(mysqlContainer));
     }
 
     @AfterClass(alwaysRun = true)
     public final void destroy()
     {
-        mysqlServer.close();
+        mysqlContainer.close();
+    }
+
+    private static String getJdbcUrl(MySQLContainer<?> container)
+    {
+        return format("%s?user=%s&password=%s&useSSL=false&allowPublicKeyRetrieval=true",
```

Comment:
`org.testcontainers.containers.JdbcDatabaseContainer#createConnection` can do this for you

Reply:
Only if you want a connection. I actually need the URL with username/password to pass to the configuration. I wish it did have an easy way to do this.

If you don't care about the password, you could do something like this:
```java
String jdbcUrl;
try (Connection connection = container.createConnection("")) {
    jdbcUrl = connection.getMetaData().getURL().toString();
}
```
But this seems overkill to just constructing it in one line.